provider/zk.go

```diff
@@ -9,7 +9,7 @@ import (
 
 // Zookepper holds configurations of the Zookepper provider.
 type Zookepper struct {
-	Kv
+	Kv `description:"go through"`
```

Comment:
Hum why is this here and not in the other `KV` based provider ? :stuck_out_tongue_closed_eyes: